servers/rendering/renderer_rd/renderer_scene_render_rd.cpp

```diff
@@ -1424,7 +1424,7 @@ void RendererSceneRenderRD::sdfgi_set_debug_probe_select(const Vector3 &p_positi
 RendererSceneRenderRD *RendererSceneRenderRD::singleton = nullptr;
 
 bool RendererSceneRenderRD::is_vrs_supported() const {
-	return RD::get_singleton()->has_feature(RD::SUPPORTS_ATTACHMENT_VRS);
+	return RD::get_singleton()->has_feature(RD::SUPPORTS_ATTACHMENT_FRAGMENT_SHADING_RATE) || RD::get_singleton()->has_feature(RD::SUPPORTS_FRAGMENT_DENSITY_MAP);
```

Comment:
Same here, I don't like this direction. The `has_feature` check inside of the driver is perfectly capable of checking if one of these extensions is enabled. 
Splitting that outside of the driver means that this list of checks will grow as we add more and more GPU drivers that have some form of VRS support.

(and I realise that probably came from my original PR, but that wasn't smart of me and we should do better :P )